content/spin/v2/rust-components.md

```diff
@@ -298,9 +339,9 @@ components.
 Let's see how we can use the Rust SDK to connect to Redis:
 
 ```rust
-use anyhow::{Context, Result};
+use anyhow::{Context};
```

Comment:
Nit: the syntax used here is allowed, but `{}` are not normally used when only importing one thing from a module. Instead: 

```suggestion
use anyhow::Context;
```